<launch>
  <arg name="config_file" default="$(find-pkg-share pose_initializer)/config/pose_initializer.param.yaml"/>

  <node pkg="pose_initializer" exec="pose_initializer" name="pose_initializer" output="log">
    <remap from="initialpose" to="/initialpose"/>
    <remap from="initialpose3d" to="/initialpose3d"/>
    <remap from="gnss_pose_cov" to="/sensing/gnss/pose_with_covariance"/>
    <remap from="pointcloud_map" to="/map/pointcloud_map"/>
    <remap from="ndt_align_srv" to="/localization/pose_estimator/ndt_align_srv"/>
    <remap from="service/initialize_pose" to="/localization/util/initialize_pose"/>
    <remap from="service/initialize_pose_auto" to="/localization/util/initialize_pose_auto"/>
    <remap from="ekf_trigger_node" to="/localization/pose_estimator/trigger_node"/>
    <remap from="ndt_trigger_node" to="/localization/pose_twist_fusion_filter/trigger_node"/>
    <remap from="client_partial_map_load" to="/map/get_partial_pointcloud_map"/>

    <param from="$(var config_file)"/>
    <param name="enable_gnss_callback" value="true"/>
  </node>
</launch>
